packages/vue/src/DataDisplay/ListItem/components/OcPage.vue

```diff
@@ -19,7 +19,7 @@ const getPageThumbnail = (url) => {
 </script>
 
 <template>
-  <div class="w-full p-5 flex">
+  <div class="w-full p-5 flex" @mouseleave="isOpen = false">
```

Comment:
why are we hiding list item action dropdown?

Reply:
![image (4)](https://github.com/hit-pay/orchid/assets/46804248/094718d7-784d-4376-afd6-7eca8cc95903)
get issue from QA

Reply:
ok